x-pack/plugins/alerting/server/alert/alert.ts

```diff
@@ -52,7 +52,7 @@ export class Alert<
   ActionGroupIds extends string = never
 > {
   private scheduledExecutionOptions?: ScheduledExecutionOptions<State, Context, ActionGroupIds>;
-  private meta: AlertInstanceMeta;
+  private meta: MutableAlertInstanceMeta;
```

Comment:
The file does a bunch of mutations of the meta object, it felt best to continue that path for now instead of trying to fix it.